xmr-btc/src/bob.rs

```diff
@@ -373,6 +371,8 @@ impl State0 {
         let s_b = cross_curve_dleq::Scalar::random(rng);
         let v_b = monero::PrivateViewKey::new_random(rng);
 
+        let dleq_proof_s_b = cross_curve_dleq::Proof::new(rng, &s_b);
```

Comment:
@luckysori what do you think? Do you see any downsize in doing so?

Reply:
Nope, seems okay!

By the way, I would still recommend replacing this proof with Lloyd's version of it in `secp256kfun`. It's faster, smaller and more likely to be correct. Just a reminder.

Reply:
Could you open an issue to describe in a bit more details please?

Reply:
Is it okay if instead I open a simple issue and I do it quickly once you think the codebase is a bit more stable?

I can write a detailed explanation of what to do, but it feels like a waste of time.

Reply:
https://github.com/comit-network/xmr-btc-swap/issues/90

Reply:
> Is it okay if instead I open a simple issue and I do it quickly once you think the codebase is a bit more stable?
> 
> I can write a detailed explanation of what to do, but it feels like a waste of time.

Sounds great, thanks!